build.sh

```diff
@@ -17,6 +17,7 @@ function usage()
   echo "  -l|--language python(default) "
   echo "                          build native library for python"
   echo "                java      build native library for java"
+  echo "                all       build native library for java & python"
```

Comment:
Instead of providing an `all` option, I think it'd be better to make this option a list, E.g., `--language python,java`.
Because if we have more languages in the future, it's easier to extend. Users can choose any combination of the languages to build.